src/tools/tools/Logger.fs

```diff
@@ -297,6 +311,16 @@ type internal DefaultLogger () =
         member this.Log ((logId : string, logLevel : LogLevel, message : string)) =
             // DefaultLogger does not support "log id" yet
             EmitLogEntry(logLevel, message)
+        
+        member this.Log ((logId : string, jobID: Guid, logLevel : LogLevel, message : string)) =
```

Comment:
Revised. I do have a concern as the ILoggerProvider has 4 Log flavors that needed to be extended when someone implements a customized Log function. Will that be fine?